package.json

```diff
@@ -327,7 +327,6 @@
     "peerDependencies": {
         "@box/cldr-data": "^34.2.0",
         "@box/react-virtualized": "9.22.3-rc-box.2",
-        "@hapi/address": ">=2.0.0 <2.1.2",
```

Comment:
The new dependency should appear in `peerDependencies`, since it's used at runtime. This is a fix and a breaking change, so let's update the commit message and PR title as described in our [contribution](https://github.com/box/box-ui-elements/blob/master/CONTRIBUTING.md#commit-types) docs.

Reply:
Ah ha, can I ask about that? I noticed that we don't use straight up dependencies in this repo. Why is that? Is the idea that, because people may only use parts of BUIE, they might not need all of its dependencies?